mne_bids/tests/test_read.py

```diff
@@ -317,6 +317,39 @@ def test_handle_coords_reading():
     with pytest.raises(RuntimeError, match='Channels do not correspond'):
         raw_test = read_raw_bids(bids_fname, bids_root)
 
+    # make sure montage is set if there are coordinates w/ 'n/a'
+    # but not set in raw.info['bads'] as well
```

Comment:
I would say it should be set if it's present. Shouldn't depend on the good/bad status. The good/bad status is only for further processing in MNE. Should have nothing to do with MNE-BIDS, no?